<launch>
    <!-- run dual-laser localisation -->
   <node name="ndt_mcl" pkg="ndt_localization" type="ndt_mcl_node" output="screen">
      <!-- jff file with map -->
    <param name="map_file" value="$(find ndt_fuser)/maps/basement2d_map.jff"  />
    <!-- resolution of jff map -->
    <param name="resolution" value="0.6" />
    <!-- show aprticles for debug purposes -->
    <param name="show_particles" value="true" />
    <!-- particle count for ndt-mcl -->
    <param name="particle_count" value="400" />
    <param name="show_pose" value="false"/>
    <param name="force_SIR" value="false" />
    <param name="laser_topic_name" value="laserscan" />
    <param name="odom_tf" value="odom_base_link" />
    <param name="root_tf" value="world" />
  </node>

  <node name="ndt_map" pkg="ndt_localization" type="ndt_map_publisher" output="screen">
    <param name="map_file" value="$(find ndt_fuser)/maps/basement2d_map.jff"  />
    <param name="resolution" value="0.6" />
    <param name="map_rate" value="0.1" />
  </node>


</launch>

